ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/DataConnections.resource

```diff
@@ -26,12 +26,41 @@ Create S3 Data Connection
     [Arguments]    ${project_title}    ${dc_name}    ${aws_access_key}    ${aws_secret_access}
     ...            ${aws_s3_endpoint}=${S3.AWS_DEFAULT_ENDPOINT}    ${aws_region}=${S3.AWS_DEFAULT_REGION}
     ...            ${connected_workbench}=${NONE}    ${press_cancel}=${FALSE}    ${aws_bucket_name}=${NONE}
+    ...            ${dc_namespace}=${NONE}
+    ...            ${aws_s3_ca_bundle}=${NONE}
     Click Button    ${DC_ADD_BTN_1_XP}
     Fill Data Connection Form    ${project_title}    ${dc_name}    ${aws_access_key}    ${aws_secret_access}
     ...            ${S3_DC_ADD_BTN}    aws_s3_endpoint=${aws_s3_endpoint}    aws_region=${aws_region}
     ...            connected_workbench=${connected_workbench}    press_cancel=${press_cancel}
     ...            aws_bucket_name=${aws_bucket_name}
     Wait Until Project Is Open    project_title=${project_title}
+    Patch S3 Data Connection With CA Bundle
+    ...    dc_name=${dc_name}    dc_namespace=${dc_namespace}    aws_s3_ca_bundle=${aws_s3_ca_bundle}
+
+Patch S3 Data Connection With CA Bundle
+    [Documentation]    Patch S3 Data Connection With CA Bundle
+    ...    If 'dc_name' or 'dc_namespace' is missing then this keyword is skipped
+    ...    If 'ca_bundle' is missing then it is taken from global variable S3.AWS_CA_BUNDLE
```

Comment:
we don't have this global variable in the test-variable file. You need another MR for adding it (apart from changing the `ods_ci/utils/scripts/testconfig/generateTestConfigFile.py`

Reply:
That global variable is option and is added in this PR at generateTestConfigFile.py